src/main/resources/i18n/displayStrings.properties

```diff
@@ -1772,6 +1783,8 @@ If you have not fulfilled the above requirements you will lose your security dep
 to get in contact with the {1} buyer by using the provided email address or mobile number to verify that he or she \
   is really the owner of the Zelle (ClearXchange) account.
 
+payment.moneyGram.info=When using MoneyGram the BTC buyer has to send the MTCN (tracking number) and a photo of the receipt by email to the BTC seller. \
+  The receipt must clearly show the seller''s full name, country and the amount. The buyer will get displayed the seller''s email in the trade process.
```

Comment:
There is one " ' " too much in selle**r's**

Reply:
<img width="707" alt="bildschirmfoto 2018-04-27 um 15 57 27" src="https://user-images.githubusercontent.com/170962/39366268-b7cea06c-4a33-11e8-932c-97f49210e7b3.png">

Reply:
2 '' are needed when there was a variable place holder before the in that text. If not only 1 ' must be used. Property support is a bit weird here...